test/system/055-rm.bats

```diff
@@ -159,4 +159,40 @@ load helpers
     fi
 }
 
+@test "podman container rm --force doesn't leave running processes" {
+    local cname=c$(random_string 30)
+    run_podman run -d --name $cname \
+               --health-cmd /bin/false \
+               --health-interval 1s \
+               --health-retries 2 \
+               --health-timeout 1s \
+               --health-on-failure=stop \
+               --stop-timeout=2 \
+               --health-start-period 0 \
+               --stop-signal SIGTERM \
+               $IMAGE sleep infinity
+    local cid=$output
+
+    # We'll use the PID later to confirm that container is not running
+    run_podman inspect --format '{{.State.Pid}}' $cname
+    local pid=$output
+
+    for i in {1..10}; do
+        run_podman inspect $cname
```

Comment:
Why not just `inspect --format '{{.State.Status}}'`?

Reply:
yes, that is definitely better.  Thanks!